stdlib/public/core/Character.swift

```diff
@@ -398,8 +398,7 @@ extension String {
       self = String._fromWellFormedCodeUnitSequence(
         UTF8.self, input: smallUTF8)
     case let .large(value):
-      let buf = String(_StringCore(_StringBuffer(value)))
-      self = buf[buf.startIndex..<buf.index(after: buf.startIndex)]
+      self = String(_StringCore(_StringBuffer(value)))
```

Comment:
(I have no attachment to any particular set of assumptions or tradeoffs, just that they be consistent)